### DISCUSSION TITLE
formily 不考虑做个可视化搭建环境吗

### DISCUSSION THREAD

**Discussion by jonny-gg** [alibaba/formily#1550] [Q&A 1.x] [answered]
formily 不考虑做个可视化搭建环境吗？还是我一直没找到？如果刚刚开始使用formily，是建议使用V2还是V1

**Comment by RihuaChen** [comment, marked as answer]
https://designable.netlify.app/

**Reply by jonny-gg** [reply]
这个是不是略显简陋，是不是有限x-render那样的形式，这个designable讲实话还真不知道怎么用

**Reply by janryWang** [reply]
designable是通用解决方案，它提供的是一系列搭建底层能力，formily会在之上做表单搭建的封装(正在进行中)，具体代码参考 https://github.com/alibaba/formily/tree/formily_next/designable/antd
